contrib/gcppubsub/config/gcppubsub.yaml

```diff
@@ -233,20 +231,3 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 8080
-
----
-# Needed by the GCP PubSub Channel to communicate with GCP PubSub.	
```

Comment:
It might be worthwhile to leave this commented out, but still in the file. Or perhaps as a sibling istio-serviceentry.yaml file, so that people using Istio have a guide on what is needed.